index.js

```diff
@@ -49,6 +49,22 @@ function main(input) {
       process.exit(2)
     }
 
+    if (args[0] === '-v' || args[0] === '--version'){
+      const pck = fs.readFileSync('package.json')
+      const stats = fs.statSync('package.json')
```

Comment:
mtime?

Reply:
https://nodejs.org/api/fs.html#fs_class_fs_stats
```The timestamp indicating the last time this file was modified.```

Reply:
And why we want this in output?

Reply:
Also I know what is mtime. And why not ctime? I'm asking why do we need any time here at all?

Reply:
If you look at the issue referenced you will see there is something about a date that needs to be displayed, didn't use ctime because that indices when the file status was changed, I cared about the last time the package.json was modified, since there was no way to know the build date(at least nothing else came to my mind back then) I used the last time it was changed, assuming will only change if a new build is made.